client/src/components/Comments/CommentList.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Col, Row } from 'reactstrap';
+import moment from 'moment';
+
+class CommentList extends Component {
+  render() {
+    return (
+      <ul className="commentList">
+        {this.props.comments.map(comment => (
```

Comment:
Please break this down into sub component e.g create new component "CommentListItem" pass comment as prop to it then map "<CommentListItem comment={comment}/>"

Reply:
done!